### ISSUE TITLE
Netplay command line arg support

### ISSUE THREAD

**Issue by aabluedragon** [fgsfdsfgs/perfect_dark#329]
Hey, would be nice to have support for Netplay using command line arguments.
something like:
`--host 0.0.0.0:23513`
`--join 88.88.88.88:23513`
(Also level selection argument, if it doesn't exist yet).

Disclaimer:
I'm the author of NukemNet, and I wish to add support for your source port.

**Comment by fgsfdsfgs** [comment]
Yes, I thought of this before, but it's kind of a pain in the ass to get from startup state to the "combat sim menu open" state without running the game for at least a few frames. I'll see what I can do.

**Comment by megatog615** [comment]
also would need --profile <profilename>(or some similar) to set your name correctly

**Comment by fgsfdsfgs** [comment]
This has been added, though not for the names and such yet.
Hosting would look something like
```--port 27100 --maxclients 8 --host```
and joining would look something like
```--connect 127.0.0.1:27015```
By default this will skip the file select screen at the start of the game and select the first file there, but you can override that with ```--profile N```.

**Comment by megatog615** [comment]
perhaps a separate netplay ini(which will be used by a future headless dedicated server) that has all these settings in it as well, overridden by the command line

**Comment by fgsfdsfgs** [comment]
Implemented a [server query protocol](https://github.com/fgsfdsfgs/perfect_dark/wiki/Server-Query-Protocol) as well, might be of interest to anyone developing any sort of server browser tool.
